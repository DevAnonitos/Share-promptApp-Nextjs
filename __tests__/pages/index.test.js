import HomePage from "@/app/page";
import CreatePrompt from "@/app/create-prompt/page";
import MyProfile from "@/app/profile/page";
import UpdatePrompt from "@/app/update-prompt/page";
import UserProfile from "@/app/profile/[id]/page";

describe('Page Imports', () => {
    it('should import all Pages without errors', () => {
        expect(HomePage).toBeDefined();
        expect(CreatePrompt).toBeDefined();
        expect(MyProfile).toBeDefined();
        expect(UpdatePrompt).toBeDefined()
        expect(UserProfile).toBeDefined()
    });
});