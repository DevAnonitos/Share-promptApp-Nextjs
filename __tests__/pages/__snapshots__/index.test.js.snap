// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Snapshots should match HomePage snapshot 1`] = `
<DocumentFragment>
  <section
    class="w-full flex-center flex-col"
  >
    <h1
      class="head_text text-center "
    >
      Discover & Share
      <br
        class="max-md:hidden"
      />
      <span
        class="orange_gradient text-center"
      >
        Generative AI-Prompts
      </span>
    </h1>
    <p
      class="desc text-center"
    >
      PromptApp is an open-source AI prompting tool for modern world to discover, create and share creative prompts.
    </p>
    <section
      class="feed"
    >
      <form
        class="relative w-full flex-center"
      >
        <input
          class="search_input peer"
          placeholder="Search for a tag or a username..."
          required=""
          type="text"
          value=""
        />
      </form>
      <div
        class="mt-16 prompt_layout"
      />
    </section>
  </section>
</DocumentFragment>
`;
